### DISCUSSION TITLE
Anyway to centralize the configuration?

### DISCUSSION THREAD

**Discussion by edalzell** [release-drafter/release-drafter#1170] [Q&A] [answered]
I'd like to use this for all my laravel packages, and have the same config for each package.

Is there a way to centralize the configuration? I know I can call actions from a repo, but how do I tell release-drafter to use a configuration file that's in a centralized repo?

**Comment by edalzell** [comment]
I guess I could put all the configuration in the action? But the config is still needed to "trigger" this action, right?

**Comment by jetersen** [comment, marked as answer]
The default behavior without creating a config file in the repository is to read it from the user/organization's `.github` repository.

See this example https://github.com/jenkinsci/.github/blob/master/.github/release-drafter.yml
Note here there are no config https://github.com/jenkinsci/gitlab-branch-source-plugin/tree/master/.github but if you check the workflows folder it uses release drafter 

You can also explicitly use `_extends` which supports pointing to another config within the same repository or another repository entirely.

**Reply by justinmchase** [reply]
Note this doesn't work in enterprise github when your `.github` repo is set to `internal`. The default `GITHUB_TOKEN` does not have permissions to read from that repo and unless you want to create a pat which can and write all repos in the org, or create a custom pat for every repo you're out of luck.

**Reply by jetersen** [reply]
I believe GitHub documents the `.github` as an must be public.

At least according to this doc: https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file

But ya if you want a private/internal repo you need to use a different PAT or github app to authorize.
